### PR TITLE
chore: add check for open prs on release branches

### PR DESCRIPTION
**Pull request by Panaetius**
test

### CODE REVIEW

**Comment by rokroskar** [comment]
Where is the message displayed? In the status line of the check?

**Comment by Panaetius** [comment]
in the check, e.g. see https://github.com/SwissDataScienceCenter/renku/actions/runs/7987999769/job/21811672665?pr=3505

File: .github/workflows/release-test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          script: |
+            const prs = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              base: context.payload.pull_request.head.ref,
+              state: 'open'
+            })
+            const prs_filtered = prs.data.filter((p)=>!p.draft)
+            if ( prs_filtered.length > 0 ) {
```
Comment (Panaetius) [inline .github/workflows/release-test.yml:24]: test that reviewing triggers the action

**Review by rokroskar — APPROVED** [review]
🚀

**Review by rokroskar — DISMISSED** [review]
approving again

**Review by rokroskar — COMMENTED** [review]
trigger on comment?

**Review by rokroskar — APPROVED** [review]

**Pull request merged**